src/cron.ts

```diff
@@ -73,22 +73,11 @@ function unsafeRoundToMinute(ts: Date, roundUp = false): Date {
   return ts;
 }
 
-function makeJobForItem(
-  item: ParsedCronItem,
-  ts: string,
-  backfilled = false,
-): CronJob {
+function makeJobForItem(item: ParsedCronItem): CronJob {
   return {
     task: item.task,
-    payload: {
-      ...item.payload,
-      _cron: {
-        ts,
-        backfilled,
-      },
-    },
+    payload: item.payload ?? {},
```

Comment:
this can be hit - if payload is undefined/null. Then on the DB layer we insert the `_cron` object inside the payload. If it's null it produces an array of two eleemnts. If it's an object, the `_cron` value is inserted into it.